File: Smart_Refrigerator/RefrigeratorApp/app/src/main/java/dongyang/jeff/refrigeratorapp/SubActivity3.java
package dongyang.jeff.refrigeratorapp;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class SubActivity3 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sub3);
        ActionBar actionBar = getSupportActionBar();
        actionBar.hide();
    }
    public void inputButton1(View view){
        Intent intent = new Intent(getApplicationContext(),SubActivity4.class);
        startActivity(intent);
    }
    public void inputButton2(View view){
        Intent intent = new Intent(getApplicationContext(),SubActivity4.class);
        startActivity(intent);
    }
}

File: Smart_Farm/Smart_Farm_Application/MoASmFarm/app/src/main/java/com/example/navigationkt/SubActivity3.kt
package com.example.navigationkt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.NavController
import androidx.navigation.fragment.findNavController
import kotlinx.android.synthetic.main.activity_sub.*
import kotlinx.android.synthetic.main.activity_sub3.*
import kotlinx.android.synthetic.main.activity_sub3.view.*


class SubActivity3 : AppCompatActivity() {

    lateinit var navController1 : NavController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_sub3)

        navController1 = nav_fragment301.findNavController()

    }

}
